src/nyancat.c

```diff
@@ -485,7 +484,7 @@ int main(int argc, char ** argv) {
 								/* This was a response to the TTYPE command, meaning
 								 * that this should be a terminal type */
 								alarm(2);
-								strcpy(term, (char *)&sb[2]);
+								term = (char *)&sb[2];
```

Comment:
Yes, specifically because we use `sb` for both the terminal and the terminal size, and the order in which the client gives us those isn't necessarily defined.